docs/REST API/rest-api.md

```diff
@@ -8943,6 +8945,43 @@ Request (using `filters`):
 }
 ```
 
+### Get batch script summary
+
+Get statuses and host counts for a batch-run script.
+
+`GET /api/v1/fleet/scripts/batch/summary`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                                                                    |
+| ----            | ------- | ---- | --------------------------------------------                                                   |
+| batch_execution_id | integer | body | **Required**. The ID returned from a batch script run. |
+
+
+#### Example
+
+`GET /api/v1/fleet/scripts/batch/summary`
+
+##### Request body
+
+```json
+{
+  "batch_execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+
+```json
+{
+  "ran": 12345,
+  "pending": 234,
+  "error": 18
```

Comment:
@rachaelshaw couple things here:

1. I was able to get `canceled` in here; we don't have to necessarily display it in the UI at this stage but we can still document it in here
2. I think we do want to change `error` to `errored` because we have a standard `error` key in most of our API response structs representing "something went wrong with this API call"

Reply:
Makes sense, I'm cool with `errored` 👍

Reply:
K I'll update here, + there's a couple other fields in there that I think help w/ visualization:
```suggestion
  "errored": 18,
  "canceled": 2,
  "targeted": 12599,
  "script_id": 555,
  "script_name": "my-script.sh",
  "team_id": 123
```

Reply:
Confirming that `team_id` will come from the new summary endpoint and not be from the activity? Works just as well, want to make sure we're synced.

Reply:
> errored

To be clear we are not changing the UI to say "Errored" right

Reply:
> Confirming that `team_id` will come from the new summary endpoint and not be from the activity? Works just as well, want to make sure we're synced.

It looks like my PR put it in both -- I may have misread where you were asking for it.  I can back the activity one out if you don't think it's needed.

> To be clear we are not changing the UI to say "Errored" right

Correct -- no change in the UI, just in the API response

Reply:
Either place would work, and I implemented the UI to grab it from this new endpoint so all good if you want to pull it out of the activity response